src/lucky_router/fragment.cr

```diff
@@ -59,8 +59,30 @@ class LuckyRouter::Fragment(T)
   # the final step to finding the payload is to search for a matching request method
   getter method_to_payload = Hash(String, T).new
 
+  def initialize(@dynamic = false)
+  end
```

Comment:
This makes me nervous but I think you plan to extract a DynamicFragment right? If so then I think this is good for now and can be cleaned up in a later PR